src/python/phenix_apps/apps/scorch/cc/cc.py

```diff
@@ -55,20 +55,34 @@ def __run(self, stage):
 
                     self.print(f"executing command '{cmd.args}' in VM {vm.hostname}")
 
+                    script = self.__send_cmd_as_file(mm, vm.hostname, cmd.args)
+
                     if wait:
-                        results = utils.mm_exec_wait(mm, vm.hostname, cmd.args, once=once)
+                        results = utils.mm_exec_wait(mm, vm.hostname, script, once=once)
+
+                        self.print(f"command '{cmd.args}' executed in VM {vm.hostname} using cc")
+
+                        # HACK: If Windows, use presence of stderr to determine
+                        # success/failure instead of exit code. Ugh...
+                        if node.hardware.os_type.lower() == "windows":
+                            if results['stderr']:
+                                self.eprint(f"command '{cmd.args}' resulted in output to STDERR (assuming failure)")
+                                self.print(f"STDERR Output: {results['stderr']}")
+
+                                sys.exit(1)
+                        elif results['exitcode']:
+                            self.eprint(f"command '{cmd.args}' returned a non-zero exit code of '{results['exitcode']}'")
 
-                        self.print(f"command '{results['cmd']}' executed in VM {vm.hostname} using cc")
+                            if results['stderr']:
+                                self.print(f"STDERR Output: {results['stderr']}")
 
-                        if results['exitcode']:
-                            self.eprint(f"command '{results['cmd']}' returned a non-zero exit code of '{results['exitcode']}'")
                             sys.exit(1)
 
-                        self.print(f"results from '{results['cmd']}':")
-                        self.print(results['stdout'])
+                        if results['stdout']:
+                            self.print(f"STDOUT Output: {results['stdout']}")
 
                         if validator:
-                            self.print(f"validating results from '{results['cmd']}'")
+                            self.print(f"validating results from '{cmd.args}'")
 
                             tempfile = f'/tmp/{str(uuid.uuid4())}.sh'
```

Comment:
should we also add the experiment name and run name to the verification scripts?